packages/core/src/api/app/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC, useState } from 'react';
+import { GitHub } from '@material-ui/icons';
+import Page from '../../../layout/Page';
+import Header from '../../../layout/Header';
+import Content from '../../../layout/Content/Content';
+import ContentHeader from '../../../layout/ContentHeader/ContentHeader';
+import {
+  Grid,
+  Typography,
+  Button,
+  TextField,
+  List,
+  ListItem,
+  Link,
+} from '@material-ui/core';
+import InfoCard from '../../../layout/InfoCard/InfoCard';
+
+enum AuthType {
+  GitHub,
+}
+
+const LoginPage: FC<{}> = () => {
+  let [githubUsername, setGithubUsername] = useState(String);
+  let [githubPersonalAuthToken, setGithubPersonalAuthToken] = useState(String);
+  let [loginDetails, setLoginDetails] = useState(Object);
+
+  const saveGithubInfo = (info: {}) => {
+    localStorage.setItem('githubLoginDetails', JSON.stringify(info));
+    setLoginDetails(info);
+  };
+
+  const deleteGithubInfo = () => {
+    localStorage.removeItem('githubLoginDetails');
+    setLoginDetails(undefined);
+  };
+
+  const handleTokenRegistration = (event: any) => {
+    switch (event.target.name) {
+      case 'github-username-tf':
+        setGithubUsername(event.target.value);
```

Comment:
Since we're fetching the user info anyway, we don't really need to input the username right? :grin:

Reply:
Per github's doc https://developer.github.com/v3/auth/#basic-authentication, they require the username as well inorder to make the api. But i agree, it is redundant!